lib/ansiblelint/formatters/__init__.py

```diff
@@ -26,3 +26,22 @@ def format(self, match):
                                 "E" + match.rule.id,
                                 match.message,
                                 )
+
+
+try:
+    from ansible.color import stringc
+    ANSIBLE_VERSION = 1
+except ImportError:
+    from ansible.utils.color import stringc
+    ANSIBLE_VERSION = 2
```

Comment:
Sorry, I realise my response wasn't clear. Just delete the two lines that set ANSIBLE_VERSION.

I mentioned utils.py because that file does need to set ANSIBLE_VERSION. We don't need to here.